commons-geometry-euclidean/src/test/java/org/apache/commons/geometry/euclidean/twod/rotation/Rotation2DTest.java

```diff
@@ -150,20 +150,18 @@ public void testCreateRotationVector() {
         final double max = 8;
         final double step = 1;
 
-        EuclideanTestUtils.permuteSkipZero(min, max, step, (ux, uy) -> {
-            EuclideanTestUtils.permuteSkipZero(min, max, step, (vx, vy) -> {
+        EuclideanTestUtils.permuteSkipZero(min, max, step, (ux, uy) -> EuclideanTestUtils.permuteSkipZero(min, max, step, (vx, vy) -> {
```

Comment:
I do not see a need for this change. I find this new format harder to read than the previous one since it obscures the fact that this is essentially a nested for loop.

Reply:
OK. reverted